tools/editor/editor_node.cpp

```diff
@@ -1,4 +1,4 @@
-/*************************************************************************/
+﻿/*************************************************************************/
```

Comment:
That's weird indeed. Even in the raw patch I don't see any difference: https://github.com/godotengine/godot/commit/4427833746b94f35d304afa7ea47bd4976a21229.patch
Could be a line ending character that changed (LF to CRLF for example), or an added BOM marker.